index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="eng">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <Title>Portfolio</Title>
+        <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+
+    <body>
+      <div class="header">
+        <h1 class="my-logo">My Logo</h1>
+        <a class="menu-link" href="">
+          <div class="lines">
+            <div class="menu-1"></div>
+            <div class="menu-2"></div>
+            <div class="menu-3"></div>
+          </div>
+        </a>
+      </div>
+      
+      <div class="headline">
+        <h3>Hey There. I'm Anyars.</h3>
+        <h2>I'm a Software Developer</h2>
+        <p class="about-me">
+            I can help you build a product , feature or website Look through some 
+            of my work and experience! If you like what you see and have a project 
+            you need coded, don't hesitate to contact me.
+        </p>
+
+        <div class="social">
+          <a class ="init-icon" href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-Cat.png" alt="Linkedin"></a>
+          <a href="https://twitter.com/anyarsencarta"><img src="./logos/Linkedin.png" alt="Linkedin"></a>
+          <a href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-Hand.png" alt="Linkedin"></a>
+          <a href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-Bird.png" alt="Twitter"></a>
+          <a href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-M.png" alt="Linkedin"></a>
+        </div>
```

Comment:
Excellent work adding an `alt` attribute to each `img` 💯 

- [ ] As was requested before with the list of navigation items, this also is a list of items! So, let's implement a `ul` and `li` with `a` inside to follow best practices 🥇